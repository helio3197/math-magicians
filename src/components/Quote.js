import React from 'react';

const Quote = () => (
  <>
    <h2
      style={{
        display: 'flex',
        flexDirection: 'column',
        margin: '0',
        padding: '24px calc(6px + 5vw)',
      }}
    >
      <q
        style={{
          marginBottom: '16px',
          fontSize: '2rem',
        }}
      >
        Pure mathematics is, in its way, the poetry of logical ideas.
      </q>
      <i
        style={{
          alignSelf: 'end',
          color: '#8d8787',
        }}
      >
        â€” Albert Einstein, German theoretical physicist
      </i>
    </h2>
  </>
);

export default Quote;
