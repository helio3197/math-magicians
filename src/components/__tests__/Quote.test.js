import { render } from '@testing-library/react';
import Quote from '../Quote';

describe('Test for Quote component', () => {
  test('The component should render', () => {
    const { asFragment } = render(<Quote />);

    expect(asFragment()).toMatchSnapshot();
  });
});
