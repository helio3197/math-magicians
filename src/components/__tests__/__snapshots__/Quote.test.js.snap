// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Quote component The component should render 1`] = `
<DocumentFragment>
  <h2
    style="display: flex; flex-direction: column; margin: 0px;"
  >
    <q
      style="margin-bottom: 16px; font-size: 2rem;"
    >
      Pure mathematics is, in its way, the poetry of logical ideas.
    </q>
    <i
      style="align-self: end; color: rgb(141, 135, 135);"
    >
      â€” Albert Einstein, German theoretical physicist
    </i>
  </h2>
</DocumentFragment>
`;
